spk/squidguard/src/installer.sh

```diff
@@ -37,28 +37,40 @@ postinst ()
     # Patch template files
     hostname=`hostname`
     sed "s/==HOSTNAME==/$hostname/g" ${ETC_DIR}/squidguard.conf.tpl > ${ETC_DIR}/squidguard.conf
-    
+    sed "s/==HOSTNAME==/$hostname/g" ${ETC_DIR}/squidclamav.conf.tpl > ${ETC_DIR}/squidclamav.conf    
     # Correct the files ownership
     chown -R ${RUNAS}:users ${SYNOPKG_PKGDEST}
-    chown 0:0 ${SQUID_WRAPPER}
-	chmod +s ${SQUID_WRAPPER}
-	
-    # Init squid cache directory
-    su - ${RUNAS} -c "${SQUID} -z -f ${CFG_FILE}"
+
+    if [ "${SYNOPKG_PKG_STATUS}" == "INSTALL" ]; then
+        # Init squid cache directory
+        su - ${RUNAS} -c "${SQUID} -z -f ${CFG_FILE}"
```

Comment:
Please remove the `-` after su, throughout this file (and the whole PR really ;))